notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcWithdrawalIntegrationTest.kt

```diff
@@ -717,6 +720,43 @@ class BtcWithdrawalIntegrationTest {
         environment.transactionHelper.addToBlackList(btcAddressDest)
     }
 
+    /**
+     * Note: Iroha and bitcoind must be deployed to pass the test.
+     * @given two registered BTC clients. 1st client has 1 BTC in wallet. Fee rate was not set.
+     * @when 1st client sends SAT 2000 to 2nd client
+     * @then transaction fails, because fee rate was not set
+     */
+    @Test
+    fun testWithdrawalFeeRateWasNotSet() {
```

Comment:
Could you also demonstrate a test with checking of the fee recalculation behavior as well?

Reply:
This is a good idea. But I don't know how to do it effectively.